### PR TITLE
Disable JUnit-style reporting for benchmark script

### PR DESCRIPTION
**Pull request by shyamjvs**
Fixes https://github.com/kubernetes/test-infra/issues/6392

/cc @porridge 

```release-note
NONE
```

### CODE REVIEW

**Comment by shyamjvs** [comment]
@wojtek-t Could you review this PR?

File: hack/jenkins/benchmark-dockerized.sh [hunk 1]
```diff
@@ -52,3 +49,5 @@ cd /go/src/k8s.io/kubernetes
 # Run the benchmark tests and pretty-print the results into a separate file.
 make test-integration WHAT="$*" KUBE_TEST_ARGS="-run='XXX' -bench=. -benchmem" \
   | tee >(prettybench -no-passthrough > ${ARTIFACTS_DIR}/BenchmarkResults.txt)
+
+sleep 100
```
Comment (wojtek-t) [inline hack/jenkins/benchmark-dockerized.sh]: 1. Why do you need this?

2. Remove this funny character at the end of line :)

**Comment by shyamjvs** [comment]
Sorry, forgot to push a change. PTAL now.

**Comment by shyamjvs** [comment]
Renamed the artifacts dir to _artifacts based on https://github.com/kubernetes/test-infra/issues/6392#issuecomment-362007934.
Reapplying lgtm .

**Pull request merged**